pkg/util/template.go

```diff
@@ -47,9 +49,9 @@ func NewTemplateFromString(name string, content string) (Template, error) {
 }
 
 // NewTemplate creates a new template for the given file
-func NewTemplate(fileName string) (Template, error) {
+func NewTemplate(fs afero.IOFS, fileName string) (Template, error) {
 
-	templ, err := template.ParseFiles(fileName)
+	templ, err := template.ParseFS(fs, fileName)
```

Comment:
This allows to mock the template.Parse method using a virtual filesystem. Useful for integration test